### DISCUSSION TITLE
Switching depName/packageName for regex manager

### DISCUSSION THREAD

**Discussion by wwuck** [renovatebot/renovate#28813] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab self-hosted Renovate 37.334.1

### Please tell us more about your question or problem

I saw the issue https://github.com/renovatebot/renovate/issues/16012 and also seeing that the [matchDepNames](https://docs.renovatebot.com/configuration-options/#matchdepnames) config says `For advanced use only! Use at your own risk!`, I thought I would try changing all `matchDepNames` to `matchPackageNames` in my renovate config and change all references of dep names to package names.

When I try to change the regex manager, I get a validation error that regex manager doesn't support `packageName` or `packageNameTemplate`.  Is this going to change, or will regex manager continue to only support the advanced config `depName` and `depNameTemplate`?

I also saw https://github.com/renovatebot/renovate/discussions/28753 but it doesn't really explain why `matchDepNames` is considered advanced and should be avoided in preference of using `matchPackageNames`.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Docs for matchDepNames will be changed/fixed in https://github.com/renovatebot/renovate/pull/28830

packageNameTemplate should be supported: https://docs.renovatebot.com/configuration-options/#packagenametemplate. Please create a reproduction repo if you see different.

**Reply by rarkins** [reply, marked as answer]
Never mind, I figured out what you meant and will fix it
